madmom/features/tempo.py

```diff
@@ -512,6 +726,7 @@ def write_tempo(tempi, filename, mirex=False):
     # format as a numpy array
     out = np.array([t1, t2, strength], ndmin=2)
     # write to output
-    np.savetxt(filename, out, fmt='%.2f\t%.2f\t%.2f')
+    if filename is not None:
+        np.savetxt(filename, out, fmt='%.2f\t%.2f\t%.2f')
```

Comment:
Isn't the visualize stuff [in line 411+ in tempo.py exactly for this](https://github.com/SebastianPoell/madmom/blob/master/madmom/features/tempo.py#L411)? At least it is how [I did it for beats](https://github.com/CPJKU/madmom/blob/master/madmom/features/beats.py#L1124).